slycot/analysis.py

```diff
@@ -1838,7 +1838,7 @@ def ab13md(Z, nblock, itype, x=None):
     return bound, d, g, x[:m+mr-1]
 
 
-def ag08bd(l,n,m,p,A,E,B,C,D,equil='N',tol=0.0,ldwork=None):
+def ag08bd(l,n,m,p,A,E,B,C,D,equil='N',tol=0.0,ldwork=None): # noqa: E741
```

Comment:
Just switch off E741 in a global option in pyproject.toml

```toml
[tool.ruff.lint]
ignore = [ "E741" ]
```

Reply:
Done. If it weren't backward incompatible I'd be tempted to change `l` to `ell`, but so it goes.